src/Service/Logger.service.js

```diff
@@ -130,7 +178,7 @@ export default class LoggerService extends DependencyAwareClass {
       Epsagon.setError(error);
     }
 
-    this.logger.log('error', message, { error });
+    this.logger.log('error', message, { error: this.processMessage(error) });
```

Comment:
I think the naming is a bit confusing. We have a method that takes `error` and `message`, but its `error` that we pass to another method called `processMessage`

Reply:
I agree. How would you name the method?

Reply:
🤣 erm .... well ... maybe just have a processError method. And check it's an error.

Like, would we ever want to process a string really?

Reply:
Yeah, am aware of that ... was envisaging the check would happen there as well.